include/deal.II/opencascade/utilities.h

```diff
@@ -142,7 +142,7 @@ namespace OpenCASCADE
    * mesh it automatically. We remark that the automatic mesh generation in
    * OpenCASCADE takes care only of the geometrical resemblance between the
    * shape and the mesh, to control the shape and regularity of the triangles
-   * you should use other meshing softwares. The two arguments `deflection` and
```

Comment:
I am ambivalent about this change but 'softwares' is deliberate and not a typo - people pluralize this word.

Reply:
It's usually non-english speaking folk that pluralize it probably due to the fact that their native language has a plural for it.

Reply:
Reverted in [feecc51](https://github.com/dealii/dealii/pull/14168/commits/feecc510b5e6b4b5666368ab2a85a777e12a9d8a)